app/questionnaire/questionnaire_store_updater.py

```diff
@@ -247,6 +247,11 @@ def update_section_status(
             status, section_id, list_item_id
         )
 
+    def is_section_complete(
```

Comment:
No need for this duplicate method.

Reply:
I [explained above](https://github.com/ONSdigital/eq-questionnaire-runner/pull/1106#discussion_r1231922361) in Rhys' comment the rationale for creating the forwarding method. 
Let me know what you think given that justification.

Reply:
Will fix in the way Rhys identified.

Reply:
Removed 👍